utils.sh

```diff
@@ -212,6 +212,10 @@ function node_map_to_install_config_hosts() {
 
     for ((idx=$start_idx;idx<$(($1 + $start_idx));idx++)); do
       name=$(node_val ${idx} "name")
+      ipv4_addr=$(node_val ${idx} "static_ipv4")
+      ipv4_addr_prefix=$(node_val ${idx} "static_ipv4_prefix")
+      ipv6_addr=$(node_val ${idx} "static_ipv6")
+      ipv6_addr_prefix=$(node_val ${idx} "static_ipv6_prefix")
```

Comment:
If we don't use metal3-dev-env we could use the [nth_ip function](https://github.com/openshift-metal3/dev-scripts/blob/master/network.sh#L3) instead here?